v3/meletrix/zoom98/zoom98.json

```diff
@@ -0,0 +1,305 @@
+{
+  "name": "Zoom 98",
+  "vendorId": "0x36B5",
+  "productId": "0x27D6",
+  "keycodes": ["qmk_lighting"],
+  "menus": ["qmk_rgb_matrix"],
```

Comment:
https://github.com/qmk/qmk_firmware/pull/22120

Please use this as a reference. Your current QMK firmware does not have ALL animations enabled to use `qmk_rgb_matrix`